### ISSUE TITLE
Add a speed option or key to the Snake plugin

### ISSUE THREAD

**Issue by vlasovskikh** [vlasovskikh/intellij-snake#2]
Björn Kautler:

> The snake plugin is a pretty little time-waster now that it runs again.
> But the snake is reeeeeally slow.
> Can its speed be a bit increased or made configurable please? :-)
> Or maybe increase the speed if + is pressed or similar.

The original issue [IDEA-194824](https://youtrack.jetbrains.com/issue/IDEA-194824).

**Comment by vlasovskikh** [comment]
Sounds good! I also find it a bit slow now. A pull request is welcome! I'm not sure if I will fix it by myself any time soon.

**Comment by Vampire** [comment]
You could have mentioned there is already A and Z to speed up and down.
Actually to use it I as intended I need to switch my keyboard layout, as here Z and Y are swapped, but that's ok :-)

**Comment by vlasovskikh** [comment]
@Vampire I forgot about this feature since I implemented it in 2015 :) There is also `d` for debug that shows the grid for obstacle and yummy characters.

**Comment by Vampire** [comment]
Yep, seen it in the code, and actually already discovered it accidentally :-)
